WooCommerce/src/main/kotlin/com/woocommerce/android/ui/orders/details/editing/address/AddressViewModel.kt

```diff
@@ -81,61 +84,88 @@ class AddressViewModel @Inject constructor(
         viewState = ViewState()
     }
 
-    private fun getCountryNameFromCode(countryCode: String): String {
+    private fun getCountryNameFromCode(countryCode: LocationCode): String {
         return countries.find { it.code == countryCode }?.name ?: countryCode
     }
 
-    private fun getCountryLocationFromCode(countryCode: String): Location {
+    private fun getCountryLocationFromCode(countryCode: LocationCode): Location {
         return Location(countryCode, getCountryNameFromCode(countryCode))
     }
 
-    private fun getStateNameFromCode(stateCode: String): String {
-        return states.find { it.code == stateCode }?.name ?: stateCode
-    }
-
-    private fun getStateLocationFromCode(stateCode: String): Location {
-        return Location(stateCode, getStateNameFromCode(stateCode))
+    private fun getStateLocationFromCode(countryCode: LocationCode, stateCode: LocationCode): Location {
+        return Location(
+            code = stateCode,
+            name = dataStore.getStates(countryCode).firstOrNull { state -> state.code == stateCode }?.name ?: stateCode
+        )
```

Comment:
small np: it may be redundant to have the declared return type followed by `return Location`, maybe we could just use the `=`return operator using inferred type for the function to reduce verbosity? 
```suggestion
    private fun getStateLocationFromCode(countryCode: LocationCode, stateCode: LocationCode)
        = Location(
            code = stateCode,
            name = dataStore.getStates(countryCode).firstOrNull { state -> state.code == stateCode }?.name ?: stateCode
        )
```

Reply:
Sure! Fixed in e7fd9bd6b21a6b1e5e3da47281c7dc2e665cf0dc